src/domains/chart/components/snapshot-loader.tsx

```diff
@@ -50,7 +50,7 @@ const SnapshotLoader = ({
       id: chartUuid,
     }))
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, []) // todo fetch based on state
+  }, [attributes]) // todo fetch based on state
```

Comment:
what is the reason behind this? I intentionally set it to run only once, is it not working properly?

Reply:
Removed! I had it there based on the rest :
https://github.com/netdata/dashboard/pull/126#discussion_r492636656